.github/workflows/diff-change-to-dist.yaml

```diff
@@ -14,20 +14,28 @@ jobs:
     # Abort if run from a fork, as token won't have write access to post comment
     if: github.event.pull_request.head.repo.full_name == github.repository
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0 # Need to also checkout the base branch to compare
-      - uses: actions/setup-node@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: npm
```

Comment:
I don't know if it makes sense to use the cache here – as I understand it, this'll cause `setup-node` to use the hash of the lock file to restore the dependencies we'd need for `npm install`?

However, in this workflow we don't run `npm install` and we don't use any of the project's dependencies – we use npm to globally install `js-beautify` and use it to diff the `dist` folder against the main branch.

Reply:
I used to think this too!

Using `cache: npm` via `actions/setup-node` will cache **~/.npm**

For example, when **./node_modules** is missing (or cleared via `npm ci`) it avoids costs network traffic to https://registry.npmjs.com again (which was slowing the tests down)

You can see your own location here:

```console
npm config get cache
```